src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -141,7 +141,7 @@
 #define fmt_VERSION 8.0.1
 
 /// Version information for python
-#define Python3_VERSION 3.8.5
+/* #undef Python3_VERSION */
```

Comment:
I have the same modification in my PRs. I never know if we should commit this file or not?

Reply:
probably not but I always do `git add -u` so it added it automatically.

Reply:
For any config macros that are platform or build type dependent, we typically "fix" them to particular values such that this header is never modified by the build:
https://github.com/GEOSX/GEOSX/blob/develop/src/cmake/GeosxConfig.cmake#L56-L64

We forgot to do this for all the newly added version variables. Most of our dependency version are fixed for most builds, but some, like Python, are system dependent.
 
You could either add `set(Python3_VERSION "3.8.5")` or `unset(Python3_VERSION)` as a workaround. In the first case, you should also commit this file (once and for all). Or there should be a separate PR to fix this for all variables.